starport/services/network/join.go

```diff
@@ -0,0 +1,218 @@
+package network
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	launchtypes "github.com/tendermint/spn/x/launch/types"
+	"github.com/tendermint/starport/starport/pkg/events"
+)
+
+// Join creates the RequestAddValidator message into the SPN
+func (b *Builder) Join(
```

Comment:
If think `Join` should include the logic to add the account and the validators since this is the name of the command. And include account and validator underlying routines inside it